app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -90,10 +96,16 @@ export default {
   },
   mounted() {
     // Fetch API Call
-    this.$store.dispatch('auditlogs/fetch', { page: 1 });
     this.$store.dispatch('agents/get');
   },
   methods: {
+    fetchAuditLogs() {
+      this.$store.dispatch('auditlogs/fetch', { page: 1 }).catch(error => {
```

Comment:
```suggestion
      this.$store.dispatch('auditlogs/fetch', { page: 1 }).catch(error => {
```

The page number, probably should be retrieved from the URL before fetching. Given we already saved the page number in the URL

Reply:
@scmmishra Thank you for catching this 🙇